Gemfile

```diff
@@ -10,6 +10,4 @@ gemspec
 # Git. Remember to move these dependencies to your gemspec before releasing
 # your gem to rubygems.org.
 
-ruby '~> 2.6.2'
```

Comment:
Removing this because the ruby version is either going to be set by your ruby version manager locally or the .travis.yml for our CI runs